pxtblocks/blocklyloader.ts

```diff
@@ -842,8 +969,8 @@ namespace pxt.blocks {
             // In addition to the user's variables, we also want to display the default
             // variable name at the top.  We also don't want this duplicated if the
             // user has created a variable of the same name.
-            goog.array.remove(variableList, Blockly.Msg.VARIABLES_DEFAULT_NAME);
-            variableList.unshift(Blockly.Msg.VARIABLES_DEFAULT_NAME);
+            goog.array.remove(variableList, "item");
+            variableList.unshift("item");
```

Comment:
I changed my mind. But, it should be `lf("_default_var")` because the word "item" is short and has different meanings in different contexts. Please add a translation to "item" in English. For non-European languages, do not translate it and choose alphabet only name for "_default_var".